### DISCUSSION TITLE
Slow on Phase 3 plotting.

### DISCUSSION THREAD

**Discussion by bryantcj89** [swar/Swar-Chia-Plot-Manager#1143] [General]
I realized one of my machine always have slow performance issue on Phase 3, 2TB NVME, 5 plots in concurrent. 

I have another machine with same setting and hardware. Performance is Ok. Only that machine having problem? Any idea? NVME issue? I tried to benchmark, speed is fast.

**Comment by racabu** [comment]
This is more like a question to Chia, not the Swar plot manager.
The manager has no control over how the plot runs... just send the initial command and monitors.

**Comment by bryantcj89** [comment]
I tried to shut down the computer, take out the nvme and put in back. then start again. It works so far. I think maybe the processor overheated or something else.

**Comment by pulltoy** [comment]
I had same issue , was a bad memory stick.    plots would fail and stall on phase 3 .   get a memtester  util and run a few loops.
